src/plugins/intel_gpu/src/plugin/ops/scatter_nd_update.cpp

```diff
@@ -20,6 +20,25 @@ static void CreateScatterNDUpdateOp(Program& p, const std::shared_ptr<ngraph::op
     std::string layerName = layer_type_name_ID(op);
     auto indices_rank = op->get_input_shape(1).size();
 
+    auto indices_constant = std::dynamic_pointer_cast<ngraph::op::Constant>(op->get_input_node_shared_ptr(1));
+    if (!indices_constant) {
+        IE_THROW() << "Unsupported parameter nodes type in " << op->get_friendly_name() << " (" << op->get_type_name() << ")";
+    }
+    else{
```

Comment:
I think we should only throw execption when "index tensor is constant && the indices values are not valid" 
otherwise let it be.

So I mean, line 25 is not needed, and it can be 
if (indices_constant) {
   // checking indices...
   if invalid, return
}

Reply:
Fixed :)